### PR TITLE
MNT: Fix configparser + python 2 deprecations

### PR DESCRIPTION
**Pull request by jond01**
Refactor python 2 prints and imports
Also some style

### CODE REVIEW

File: nibabel/tests/test_round_trip.py [hunk 1]
```diff
@@ -181,14 +181,14 @@ def check_arr(test_id, V_in, in_type, out_type, scaling_type):
...
-               rel_mx_e,
-               slope, inter)
+        print(test_id,
+              np.dtype(in_type).str,
+              np.dtype(out_type).str,
+              exp_abs_mx_e,
+              abs_mx_e,
+              rel_thresh,
+              rel_mx_e,
+              slope, inter)
```
Comment (effigies) [inline nibabel/tests/test_round_trip.py:191]: Looks like this was supposed to print a tuple but got lucky in 2-3 conversion.

```suggestion
        print((test_id,
               np.dtype(in_type).str,
               np.dtype(out_type).str,
               exp_abs_mx_e,
               abs_mx_e,
               rel_thresh,
               rel_mx_e,
               slope, inter))
```

File: versioneer.py [hunk 2]
```diff
@@ -276,11 +276,7 @@
 
 """
 
-from __future__ import print_function
-try:
-    import configparser
-except ImportError:
-    import ConfigParser as configparser
+import configparser
```
Comment (effigies) [inline versioneer.py:279]: While these are valid updates, I'm disinclined to change versioneer except for functional fixes, to keep any diffs focused.

File: nibabel/tests/test_round_trip.py (hunk 1 above)
Comment (jond01) [reply nibabel/tests/test_round_trip.py:191]: Nice, I thought it's just python 3 with a space after the print (`print ('hi')` works)

File: versioneer.py (hunk 2 above)
Comment (jond01) [reply versioneer.py:279]: Ok, what about the changes after the imports? They are deprecated in py3.6

Comment (effigies) [reply versioneer.py:279]: Fair enough. Let's go ahead with these changes, then.

**Comment by effigies** [comment]
LGTM. Thanks!

**Pull request merged**